pom.xml

```diff
@@ -1305,6 +1305,62 @@
       </build>
     </profile>
 
+    <profile>
+      <id>jdk-21</id>
```

Comment:
what about sbt?

Reply:
sbt will respect maven's profile @cfmcgrady

Reply:
Discussed with @cfmcgrady , sbt doesn't support error prone yet, and no need to set java version. @cfmcgrady could you explain more?

Reply:
I was looking into this @pan3793 , but unfortunately could not make progress due to lack of support.
See CELEBORN-1080

Reply:
@mridulm in addition to SBT plugin, is it possible to be achieved through shell command, or POM-less Maven command? for example: https://github.com/maven-download-plugin/maven-download-plugin/issues/262

Reply:
I did not try invoking maven from within sbt - but if the goal is to move to sbt (and eventually get rid of maven), that is counterproductive to that, no ?

Reply:
we can delegate some tasks to the shell scripts. for example, we use a shell script to create the binary tarball, instead of maven/sbt assembly plugin.